generator/internal/descriptor_utils.cc

```diff
@@ -632,11 +632,29 @@ bool CheckParameterCommentSubstitutions() {
   return all_substitutions_used;
 }
 
+/// If a service name mapping exists, return the new name.
+/// Parses a command line argument in the form:
+/// {"service_name_mappings": "service_a:new_service_a,service:new_service"}.
```

Comment:
Might I suggest using the "k1=v1,k2=v2,..." syntax for a "map string" (i.e., `=` instead of `:`).  I believe it is more common, and more readable.

Reply:
That makes sense.

Made an issue to track https://github.com/googleapis/google-cloud-cpp/issues/12935